test/types.js

```diff
@@ -218,11 +218,12 @@ defineTypeTest('float', [
 });
 
 defineTypeTest('decimal', [
-  'DECIMAL(30, 10) NULL'
+  'DECIMAL(30, 10) NULL',
+  'DECIMAL(30, 20) NULL'
 ], [
   [1.0], [-1.0], [123.456], [-13.47],
```

Comment:
When you add another column to the type test table, each row array needs to contain 2 entires. We don't test the full range of specified decimal places in the current test anyways and using the same values does cover what this fixes. If you add this map function at the end of the values array, it should pass.

```
[
  [1.0], [-1.0], [123.456], [-13.47],
  [123456789.123], [-123456789.123], [null],
  [1447410019.012], [123.00000123], [0.0004321]
].map(function(x) { return [ x[0], x[0] ]; });
```